workers/unity/Packages/io.improbable.gdk.buildsystem/.codegen/Source/Generators/BuildSystem/UnityWorkerMenuGenerator.cs

```diff
@@ -0,0 +1,96 @@
+                        });
+
+                        for (var i = 0; i < workerTypes.Count; i++)
+                        {
+                            var workerType = workerTypes[i];
+                            var workerTypeString = $@"""{workerType}""";
+
+                            buildWorkerMenu.Method($"public static void BuildLocal{workerType}()", m =>
+                            {
+                                m.Annotate($@"MenuItem(EditorConfig.ParentMenu + ""/"" + LocalMenu + ""/{workerType}"", false, EditorConfig.MenuOffset + {i})");
+                                m.Line($@"MenuBuildLocal(new[] {{ {workerTypeString} }});");
+                            });
+
+                            buildWorkerMenu.Method($"public static void BuildCloud{workerType}()", m =>
+                            {
+                                m.Annotate($@"MenuItem(EditorConfig.ParentMenu + ""/"" + CloudMenu + ""/{workerType}"", false, EditorConfig.MenuOffset + {i})");
+                                m.Line($@"MenuBuildCloud(new[] {{ {workerTypeString} }});");
+                            });
+                        }
+
+                        buildWorkerMenu.Method("public static void BuildLocalAll()", m =>
+                        {
+                            m.Annotate($@"MenuItem(EditorConfig.ParentMenu + ""/"" + LocalMenu + ""/All workers"", false, EditorConfig.MenuOffset + {workerTypes.Count})");
+                            m.Line("MenuBuildLocal(AllWorkers);");
+                        });
+
+                        buildWorkerMenu.Method("public static void BuildCloudAll()", m =>
+                        {
+                            m.Annotate($@"MenuItem(EditorConfig.ParentMenu + ""/"" + CloudMenu + ""/All workers"", false, EditorConfig.MenuOffset + {workerTypes.Count})");
+                            m.Line("MenuBuildCloud(AllWorkers);");
+                        });
+
+                        buildWorkerMenu.Method("public static void Clean()", m =>
+                        {
+                            m.Annotate($@"MenuItem(EditorConfig.ParentMenu + ""/Clean all workers"", false, EditorConfig.MenuOffset + {workerTypes.Count})");
+                            m.Line("MenuCleanAll();");
+                        });
+
+                        buildWorkerMenu.Method("private static void MenuBuildLocal(string[] filteredWorkerTypes)", m =>
+                        {
+                            m.Line("WorkerBuilder.MenuBuild(BuildEnvironment.Local, filteredWorkerTypes);");
+                        });
+
+                        buildWorkerMenu.Method("private static void MenuBuildCloud(string[] filteredWorkerTypes)", m =>
+                        {
+                            m.Line("WorkerBuilder.MenuBuild(BuildEnvironment.Cloud, filteredWorkerTypes);");
+                        });
+
+                        buildWorkerMenu.Method("private static void MenuCleanAll()", m =>
```

Comment:
Would be nice to have an overload for these to straight up pass in an `IEnumerator<string>` or a delegate that returns one.

This lets you write the following:

```
buildWorkerMenu.Method("private static void MenuCleanAll()", new[]
{
    "WorkerBuilder.Clean();",
    "Debug.Log(\"Clean completed\");"
});

buildWorkerMenu.Method("private static void MenuCleanAll()", () => new[]
{
    "WorkerBuilder.Clean();",
    "Debug.Log(\"Clean completed\");"
})
```